lib/application.js

```diff
@@ -202,23 +202,39 @@ class Application extends api.events.EventEmitter {
     });
   }
 
+  loadSchemas(callback) {
+    const dir = api.path.join(this.dir, 'schemas');
+    const loaderPath = api.path.join(dir, 'config.js');
+    impress.createScript(this, loaderPath, (err, getMsConfig) => {
+      if (err) {
+        callback(err);
+        return;
+      }
+
+      const { options, config } = getMsConfig();
+      options.context = this;
+
+      api.metaschema.fs.load(dir, options, config).then(([errors, schema]) => {
+        if (errors.length) {
+          console.error(errors);
+          return;
+        }
+        this.schema = schema;
+        callback(null);
+      }, error => {
+        callback(error);
+      }).catch(callback);
```

Comment:
```suggestion
      });
```
No need to `catch` here, all errors will be passed to second function in `then`

Reply:
This catch will be used if `then` function throws an error.